IPython/core/page.py

```diff
@@ -309,7 +309,7 @@ def get_pager_cmd(pager_cmd=None):
             pager_cmd = default_pager_cmd
     
     if pager_cmd == 'less' and '-r' not in os.environ.get('LESS', ''):
```

Comment:
Capital `-R` here, I think. Or maybe we should check for both, since they're very similar in effect:

```python
'-r' not in os.environ.get('LESS', '').lower()
```